pipeline/processor.go

```diff
@@ -197,11 +199,16 @@ func (p *processor) doActions(event *Event) (isPassed bool) {
 
 		p.actionWatcher.setEventBefore(index, event)
 
-		switch action.Do(event) {
-		case ActionPass:
+		result := action.Do(event)
+		switch result {
+		case ActionPass, ActionSpawned:
 			p.countEvent(event, index, eventStatusPassed)
 			p.tryResetBusy(index)
 			p.actionWatcher.setEventAfter(index, event, eventStatusPassed)
+
+			if result == ActionSpawned {
```

Comment:
is it useless ?

Reply:
In `ActionPass` case, we will continue process this (Parent) event in the next actions :)

My idea was that we should not continue to process parent events